http-requests-core/src/main/java/com/budjb/httprequests/converter/EntityConverterManager.java

```diff
@@ -36,15 +37,17 @@
     /**
      * List of registered entity converters.
      */
-    private final List<EntityConverter> converters = new ArrayList<>();
+    private final List<EntityConverter> converters;
 
-    /**
-     * Adds an entity converter to the manager.
-     *
-     * @param converter Converter to add to the manager.
-     */
-    public void add(EntityConverter converter) {
-        converters.add(converter);
+    public EntityConverterManager(List<EntityConverter> entityConverters) {
```

Comment:
FYI, this was a somewhat breaking change.

It may also be convenient to use an array form as well, so that the following syntax could be used:

```java
EntityConverterManager entityConverterManager = new EntityConverterManager(
    new StringEntityReader(),
    new StringEntityWriter()
);
```

Without an array-form, it forces the user to have to use an intermediate ArrayList (at minimum) which can be a little cumbersome.